riscv/tests/riscv_data/many_chunks/src/main.rs

```diff
@@ -10,6 +10,10 @@ pub fn main() {
     let mut foo = Vec::new();
     foo.push(1);
 
+    for _ in 0..100 {
+        foo.push(foo.iter().sum());
+    }
```

Comment:
This was needed to cause the bug.

Reply:
Should we keep this? It's kind of random, I think it could go.

Reply:
I think we should keep it because it's what caused the bug